spec/factories/stolen_records.rb

```diff
@@ -1,11 +1,17 @@
 FactoryBot.define do
   factory :stolen_record do
-    bike { FactoryBot.create(:bike) }
+    bike { FactoryBot.create(:bike, :with_image) }
```

Comment:
Does this mean that every bike factory now adds an image by default? If so, why do that rather than having it be a separate factory? Does this have a negative impact on test speed?